htsget-search/Cargo.toml

```diff
@@ -52,11 +53,10 @@ serde = "1.0"
 tempfile = "3.3"
 data-url = "0.2"
 
-# Aws S3 storage.
-aws-credential-types = { version = "0.54", features = ["test-util"] }
-s3s = { version = "0.3" }
-s3s-fs = { version = "0.3" }
-s3s-aws = { version = "0.3" }
+# S3 storage testing
+s3s = { git="https://github.com/Nugine/s3s", version = "0.5.0-dev" }
```

Comment:
Hey @Nugine, can you push 0.5 to crates.io? Thanks for your crates again! ;)

Reply:
Ok. Released just now.

Reply:
Awesome, thanks!